shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -9,86 +9,106 @@
+    public static final String CONTAINER_APPEND_OFFSET_COUNT = "segmentstore.container.append_offset_count";
+    public static final String CONTAINER_UPDATE_ATTRIBUTES_COUNT = "segmentstore.container.update_attributes_count";
+    public static final String CONTAINER_GET_ATTRIBUTES_COUNT = "segmentstore.container.get_attributes_count";
+    public static final String CONTAINER_READ_COUNT = "segmentstore.container.read_count";
+    public static final String CONTAINER_GET_INFO_COUNT = "segmentstore.container.get_info_count";
+    public static final String CONTAINER_CREATE_SEGMENT_COUNT = "segmentstore.container.create_segment_count";
+    public static final String CONTAINER_DELETE_SEGMENT_COUNT = "segmentstore.container.delete_segment_count";
+    public static final String CONTAINER_MERGE_SEGMENT_COUNT = "segmentstore.container.merge_segment_count";
+    public static final String CONTAINER_SEAL_COUNT = "segmentstore.container.seal_count";
+    public static final String CONTAINER_TRUNCATE_COUNT = "segmentstore.container.truncate_count";
+    public static final String PROCESS_OPERATIONS_LATENCY = "segmentstore.process.operations_latency";
+    public static final String PROCESS_OPERATIONS_BATCH_SIZE = "segmentstore.process.operations_batch_size";
+    public static final String OPERATION_QUEUE_SIZE = "segmentstore.operation.queue_size";
+    public static final String OPERATION_PROCESSOR_IN_FLIGHT = "segmentstore.operation.processor_in_flight";
+    public static final String OPERATION_QUEUE_WAIT_TIME = "segmentstore.operation.queue_wait_time";
+    public static final String OPERATION_PROCESSOR_DELAY_MILLIS = "segmentstore.operation.processor_delay_ms";
+    public static final String OPERATION_COMMIT_LATENCY = "segmentstore.operation.commit_latency_ms";
+    public static final String OPERATION_LATENCY = "segmentstore.operation.latency_ms";
+    public static final String OPERATION_COMMIT_METADATA_TXN_COUNT = "segmentstore.operation.commit_metadata_txn_count";
+    public static final String OPERATION_COMMIT_MEMORY_LATENCY = "segmentstore.operation.commit_memory_latency_ms";
+    public static final String OPERATION_LOG_SIZE = "segmentstore.operation.log_size";
+    public static final String ACTIVE_SEGMENT_COUNT = "segmentstore.active_segments";
+
+    // Thread pool metrics
+    public static final String THREAD_POOL_QUEUE_SIZE = "segmentstore.thread_pool.queue_size";
+    public static final String THREAD_POOL_ACTIVE_THREADS = "segmentstore.thread_pool.active_threads";
 
     // Metrics in Controller
     // Stream request counts (Static)
-    public static final String CREATE_STREAM = "stream_created";    // Histogram
-    public static final String SEAL_STREAM = "stream_sealed";       // Histogram
-    public static final String DELETE_STREAM = "stream_deleted";    // Histogram
+    public static final String CREATE_STREAM = "controller.stream.created";    // Histogram
```

Comment:
even though these are metrics in controller, since they are stream specific metrics that user is interested in, they dont care where or how these are collected. 
So the same controller in the metrics would seem inappropriate.